src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-modals.cypress.spec.js

```diff
@@ -0,0 +1,176 @@
+import { PROFILE_PATHS } from '../../constants';
+
+import mockUser from '../fixtures/users/user-36.json';
+import mockFeatureToggles from '../fixtures/feature-toggles.json';
+
+const setup = (mobile = false) => {
+  window.localStorage.setItem(
```

Comment:
Now that we've done this _many_ times I wonder if we could make it part of the `login()` helper. @U-DON  do you have any thoughts on that?

Reply:
You mean dismissing all announcements (setting it to `'*'`)? I think we can add that as a default for all tests in general (without calling `login`); we could add it to the support file, where we have other defaults like empty maintenance windows and feature toggles. Seems like the announcements should be "opt in" if anyone wants to test something related to that. Does that make sense?

Reply:
I meant just setting this particular value for `DISMISSED_ANNOUNCEMENTS` as part of logging in since it needs to be done every time you log in. But I'd be interested in seeing how setting that value to `*` would work.